specification.md

```diff
@@ -3215,17 +3215,18 @@ end: true
 
 Parallel state defines a collection of `branches` that are executed in parallel.
 A parallel state can be seen a state which splits up the current workflow instance execution path
-into multiple ones, one for each of each branch. These execution paths are performed in parallel
+into multiple ones, one for each branch. These execution paths are performed in parallel
 and are joined back into the current execution path depending on the defined `completionType` parameter value.
 
 The "completionType" enum specifies the different ways of completing branch execution:
-* and: All branches must complete execution before state can perform its transition. This is the default value in case this parameter is not defined in the parallel state definition.
-* xor: State can transition when one of the branches completes execution
-* n_of_m: State can transition once `n` number of branches have completed execution. In this case you should also
-specify the `n` property to define this number.
+* allOf: All branches must complete execution before state can perform its transition. This is the default value in case this parameter is not defined in the parallel state definition.
+* atLeast: State can transition once a specific number of branches have completed execution. In this case you must also
+specify the `numCompleted` property to define this number.
 
 Exceptions may occur during execution of branches of the Parallel state, this is described in detail in [this section](#parallel-state-exceptions).
 
+Once the `completionType` criteria has been met for a Parallel state, the runtime implementation should make a best-effort attempt at canceling any incomplete branches before transitioning.
```

Comment:
We should have a flag for this with default set to not cancel. 

"cancelUncompletedBranches" maybe?

Cancelling branches should be explicit choice.

Reply:
When we originally discussed this idea it seemed a relatively straight-forward change was being suggested, but this isn't such a small suggestion after all.  I'm removing any mention of this from this PR.